assets/card.css

```diff
@@ -7,3 +7,7 @@
 .fixedCard:hover {
   @apply shadow-soft transition duration-200;
 }
+
+.promotionImage {
+  height: 209px;
+}
```

Comment:
this value seems to be only used for that one component. in that case, we'd prefer to use a css module. if at all possible, we try not to rely on global styles, see https://github.com/carforyou/carforyou-docs/blob/master/frontend/css-conventions.md#2-css-modules